src/components/molecules/LoginForm/LoginForm.tsx

```diff
@@ -48,10 +58,10 @@ export const LoginForm = ({navigation, route}: TLoginFormProps) => {
               placeholder="Email"
             />
           )}
-          name="mail"
+          name="email"
         />
-        {errors.mail && (
-          <CustomText variant={Variatns.ERROR} text={errors.mail.message} />
+        {errors.email && (
```

Comment:
Btw taka polecajka, do inline render polecam zrobić sobie `<If/>` component, który dostanie children, i propsa "condition", jeżeli "condition" jest truthy, to wtedy zwróci children, jeżeli nie, to zwróci null'a

Performance to nie zwiększa ani nic (boolean przez value jest zapamietywany wiec jeden chuj), ale imo jest w chuj czytelniej i wygodniej to pisać